scan/src/Colors.re

```diff
@@ -61,6 +61,8 @@ let blue5 = Css.hex("475DEB");
 let blue6 = Css.hex("243CDB");
 let blue7 = Css.hex("142AB8");
 let blue8 = Css.hex("0B1D8E");
+let blue9 = Css.hex("ABB6FF");
+let blue10 = Css.hex("000C5C");
```

Comment:
Sorry my bad, I did't tell you that colors are sorted by its lightness. So in this case blue9 should be darker than blue8.
I think we just seperate these char colors to their own place.

let chartBalanceAtStake = Css.hex("ABB6FF");
let chartReward = Css.hex("000C5C");

Please append these lines to the bottom of Colors.re